asv_bench/benchmarks/hdk/benchmarks.py

```diff
@@ -492,3 +491,6 @@ def time_groupby_agg_mean_dict(self, *args, **kwargs):
                 {col: "mean" for col in self.non_groupby_columns}
             )
         )
+
+
+from ..utils import setup  # noqa: E402, F401
```

Comment:
why we import this at the end of the file?

Reply:
I suppose not to accidentally override this function. A fairly common practice, for example, this is done in pandas.

Reply:
Example: https://github.com/pandas-dev/pandas/blob/4965c51ef63dd06bcf460165a10bffa688142c46/asv_bench/benchmarks/algorithms.py#L192

@dchigarev any objections?